f5lbaasdriver/test/tempest/tests/scenario/test_l7policies_and_rules.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from f5lbaasdriver.test.tempest.tests.scenario import f5_base
+
+import pytest
+import requests
+
+
+class TestL7PolicyBasic(f5_base.F5BaseTestCase):
+    def _remove_members(self):
+        # Due to issue 393 in github, nodes from the device are not deleted
+        # unless the members are explicitly deleted
+        self.members_client.delete_member(
+            self.pool.get('id'),
+            self.members['primary'].get('id')
+        )
+        self._wait_for_load_balancer_status(self.load_balancer.get('id'))
+        self.members_client.delete_member(
+            self.detached_pool.get('id'),
+            self.members['secondary'].get('id')
+        )
+        self._wait_for_load_balancer_status(self.load_balancer.get('id'))
+
+    def _run_traffic(
+            self, expected_server, headers=None, cookies=None,
+            expected_status=200):
+        print('##################')
```

Comment:
Yes, I'd like to leave them in for the time being, so we can sanity check the expectations against the responses.